### ISSUE TITLE
Linux 3.13.0-40-generic conflict with node version

### ISSUE THREAD

**Issue by thanhnguyen2359** [npm/npm#7182]
Hi , when i deploy today i found the issue:

```
 npm ERR! Linux 3.13.0-40-generic
       npm ERR! argv "node" "/tmp/build_8238e6eb68a3fd06d8502f85b38fd86a/.heroku/node/bin/npm" "install" "--quiet" "--userconfig" "/tmp/build_8238e6eb68a3fd06d8502f85b38fd86a/.npmrc"
       npm ERR! node v0.11.9
       npm ERR! npm  v2.1.18
       
       npm ERR! null == true
       npm ERR! 
       npm ERR! If you need help, you may report this error at:
       npm ERR!     <http://github.com/npm/npm/issues>
       npm ERR! Linux 3.13.0-40-generic
       npm ERR! argv "node" "/tmp/build_8238e6eb68a3fd06d8502f85b38fd86a/.heroku/node/bin/npm" "install" "--quiet" "--userconfig" "/tmp/build_8238e6eb68a3fd06d8502f85b38fd86a/.npmrc"
       npm ERR! node v0.11.9
       npm ERR! npm  v2.1.18
```
   Any ideas how to fix ?

**Comment by smikes** [comment]
You should probably not be running with node `0.11.9`.  If you must use the unstable version of node, please use the latest, which is `0.11.15`.  Alternatively you may wish to try the `io.js` fork of node, <https://iojs.org/>

For some Linux distributions (Debian/Ubuntu and RedHat/CentOS), the latest node version provided by the distribution may lag behind the stable version. [Here are instructions from NodeSource](https://github.com/nodesource/distributions) on getting the latest node.

Thanks!

**Comment by smikes** [comment]
Is this still a problem for you?

We are trying to clean up older npm issues, so if we don't hear back from you within a week, we will close this issue. (Don't worry -- you can always come back again and open a new issue!)

Thanks!

**Comment by thanhnguyen2359** [comment]
I have fixed by upgrade to node "0.11.14" and npm "1.3.15"

**Comment by othiym23** [comment]
Closing as resolved.
